tests/integration/test_storage_kafka/test.py

```diff
@@ -1507,8 +1507,19 @@ def produce():
     assert TSV(result) == TSV('1	1')
 
 
-@pytest.mark.timeout(600)
+@pytest.mark.timeout(90)
 def test_kafka_flush_by_block_size(kafka_cluster):
+    cancel = threading.Event()
+
+    def produce():
+        while not cancel.is_set():
+            messages = []
+            messages.append(json.dumps({'key': 0, 'value': 0}))
+            kafka_produce('flush_by_block_size', messages)
+
+    kafka_thread = threading.Thread(target=produce)
+    kafka_thread.start()
```

Comment:
This creates the topic implicitly, why not create it explicitly?

Reply:
Long story, not worth typing, just reworked test to make it closer to real-life usage (it relates to removed TODO part)